errors.ts

```diff
@@ -112,7 +115,7 @@ export class Errors implements IErrors {
 					? resolveCallStack(ex.stack)
 					: "\x1B[31;1m" + ex.message + "\x1B[0m");
 
-				if (!ex.suppressCommandHelp) {
+				if (printCommandHelp && !ex.suppressCommandHelp) {
```

Comment:
I think this change is not needed, we can use printCommandHelp as mandatory parameter.